docs/GettingStarted.mdx

```diff
@@ -20,7 +20,7 @@ import { ThemeProvider } from "styled-components";
 import { theme } from "@blasterjs/core";
 // or
 // import { MyCustomTheme } from '../your-custom-theme'
-import GlobalStyle from "@blaster/core/components/globalStyle";
+import GlobalStyle from "@blasterjs/core/components/globalStyle";
```

Comment:
Could you change this so that it groups the GlobalStyle import with the theme import? They should both be importable from `@blasterjs/core`.